### DISCUSSION TITLE
Guidance on Upgrading to Latest Long-Term Stable OpenSSL Version

### DISCUSSION THREAD

**Discussion by mahenderkk08** [openssl/openssl#24684] [Q&A] [answered]
Hello,

We are currently using OpenSSL version 3.1.2 and the FIPS module version 3.0.8 in our setup. We are looking to upgrade to the latest long-term stable version of OpenSSL.

From my research, I understand that OpenSSL 3.1.2 is currently supported until March 14, 2025 but it has some vulnerabilities.

Could you please advise on the latest stable version that we should use to ensure maximum security and stability for our environment?

Thank you for your assistance.

Regards,
Mahender K

**Comment by paulidale** [comment]
The latest LTS version is 3.0.14.

A new LTS version has to be declared at least every four years (by the [versioning policy](https://www.openssl.org/policies/general/versioning-policy.html)).  That might be slightly at odds with the [time based release policy](https://www.openssl.org/policies/general/release-policy.html), so until the next LTS is declared, 3.0.x is the safe option.

**Reply by mahenderkk08** [reply]
Hi @paulidale , Okay currently using OpenSSL version 3.1.2 and the FIPS module version 3.0.8 in our setup.  can upgrade it to 3.3.x and remain the same in using the FIPS module?

**Reply by paulidale** [reply, marked as answer]
Yes.
